lib/Doctrine/ORM/Persisters/Collection/ManyToManyPersister.php

```diff
@@ -259,6 +259,15 @@ public function loadCriteria(PersistentCollection $collection, Criteria $criteri
             . implode(' AND ', $onConditions)
             . ' WHERE ' . implode(' AND ', $whereClauses);
 
+        if ($criteria->getOrderings()) {
```

Comment:
Don't repeat this call: assign the result to a variable, then reuse the variable